src/install/doinst.sh

```
@@ -0,0 +1,3 @@
+( cd usr/local/emhttp/plugins/auto-unlock/event ; rm -rf array_started )
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)